content/source/docs/cloud/sentinel/import/tfrun.html.md

```diff
@@ -42,6 +47,58 @@ to only enforce the policy on non-development workspaces is more appropriate.
 [run-glossary]: /docs/glossary.html#run
 [workspace-glossary]: /docs/glossary.html#workspace
 
+## Namespace: root
```

Comment:
Using `root` in the docs but I am wondering if using `tfrun` is better as this is called out in the diagram above. Thoughts?

Reply:
I'm going to ping @nfagerlund incase he is aware of some prior art in our docs as I'd prioritise consistency above most else.

I _think_ (though I'd be hesitant to stake any money on this) that the semantically correct thing here isn't a root namespace. I've got a series of attributes, on the root element. So maybe we can ditch the logical grouping of them in the docs (given that grouping doesn't really exist in the object) and just list the attributes we expose? Then the two namespaces can follow after that.

Reply:
"Root" is 'correct' https://www.terraform.io/docs/cloud/sentinel/import/tfplan.html#namespace-root

I think the import docs in general could use some reformatting/naming (and that 'root' is blah) but would agree that consistency should be prioritized highly.

Reply:
Thanks for confirming @chrisarcand